administrator/components/com_joomlaupdate/src/Model/NotificationModel.php

```diff
@@ -82,17 +82,44 @@ public function sendNotification($type, $oldVersion, $newVersion): void
             'url'        => Uri::root(),
         ];
 
+        // Determine the default admin language and load the language file for the fallback and default language
+        $defaultLocale   = ComponentHelper::getParams('com_languages')->get('administrator', 'en-GB');
+        $defaultLanguage = $app->getLanguage();
+        $defaultLanguage->load('com_joomlaupdate', JPATH_ADMINISTRATOR, 'en-GB', true, true);
+        $defaultLanguage->load('com_joomlaupdate', JPATH_ADMINISTRATOR, $defaultLocale);
```

Comment:
```suggestion
        $defaultLanguage->load('com_joomlaupdate', JPATH_ADMINISTRATOR);
```
We need to load com_joomlaupdate language file of the current active language instead of default language. This fixed an issue with my test case:

- Backend default language set to German
- My super user account has Backend Language set to English
- I received email in German instead of in English as expected.

The suggested change here fixed that issue.

Reply:
Hi @joomdonation 

I have the following scenario:

Joomla backend is globally in English.
I have 4 users. 3 have super user rights and 1 user has administrator rights.
All users have a different language assigned. With the 5 different German language packs, you can set up scenarios very easily.

**Result:** All users received the email in one variant in German. This means that I cannot reproduce a scenario in which a user received an English email, even though the backend is set to en-GB globally.

Reply:
@tecpromotion The setup for my test is different :
- Only one super user account enabled
- Backend set to German
- Super user account backend language set to English

And to simplify the test, I use the testing instructions from here https://github.com/joomla/joomla-cms/pull/45721 (calling the code to send notification).

Reply:
Unfortunately, deleting `$defaultLocale` had no effect on the language of the email.
https://github.com/joomla/joomla-cms/pull/46050#issuecomment-3274757007

Reply:
Okay, I've now rewritten everything cleanly and also incorporated your change @joomdonation
Now it seems to be working.